tgui/bin/tgui-build.bat

```diff
@@ -2,4 +2,4 @@
 cd "%~dp0\.."
 call yarn install
 call yarn run build
-timeout /t 9
+timeout /t 90
```

Comment:
If you build through vscode or another terminal session this isn't an issue. It'd be better to have this remain as a lower number

Reply:
You can build TGUI through VSCODE? how?